OVERVIEW.md

```diff
@@ -148,6 +148,25 @@ jscs path[ path[...]] --preset=jquery
 
 In order to add/remove preset rules you will need to create a `.jscsrc` config file.
 
+### `--extract`
+With this option you could set file mask from which JavaScript should be extracted.
+```
+jscs path[ path[...]] --extract *.html 
+```
+
+You could set several masks if needed.
+```
+jscs path[ path[...]] --extract *.html --extract *.htm
+```
+
+Or you could you one of shortcut value: `always`, `auto` or `never`. Note that those values couldn't be combined with file masks.
```

Comment:
`you could you one of` -> `you can use a`.

`Shortcut values cannot be combined with file masks`